### PR TITLE
fix deploy karmada-descheduler.yaml

### PR DESCRIPTION
**Pull request by wongearl**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes # karmada-descheduler没有此参数，而部署文件中加了这个参数，该组件无法正常启动

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by RainbowMango** [comment]
Thanks for reporting this and providing the fix. @wongearl 

I noticed this issue this morning too:
```
-bash-4.2# kubectl logs -n karmada-system karmada-descheduler-58cf5d7c4b-44h2f 
Error: unknown flag: --karmada-kubeconfig
```

@wongearl you might have to sign your commit according to [this guideline](https://github.com/karmada-io/karmada/pull/1472/checks?check_run_id=5507383242).
You can do that after @Garrybest approves.

File: artifacts/deploy/karmada-descheduler.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ spec:
           imagePullPolicy: IfNotPresent
           command:
             - /bin/karmada-descheduler
-            - --karmada-kubeconfig=/etc/kubeconfig
```
Comment (Garrybest) [inline artifacts/deploy/karmada-descheduler.yaml:27]: Sorry, it was my mistake. Please use `--kubeconfig=/etc/kubeconfig`. 

Btw, there is another problem, I will test it again. This PR could be merged first.

**Review by Garrybest — CHANGES_REQUESTED** [review]

File: artifacts/deploy/karmada-descheduler.yaml (hunk 1 above)
Comment (wongearl) [reply artifacts/deploy/karmada-descheduler.yaml:27]: ok,thank you

**Comment by RainbowMango** [comment]
@wongearl you might have to sign your commit according to [this guideline](https://github.com/karmada-io/karmada/pull/1472/checks?check_run_id=5507383242).

**Comment by RainbowMango** [comment]
Or, you can do that by:
```
$ git reset HEAD~2
$ git commit -s -m "fix karmada-descheduler failed to start issue"
$ git push origin fix-karmada-descheduler-deploy -f 
```

**Comment by RainbowMango** [comment]
@wongearl Why closed?

**Comment by wongearl** [comment]
meet some error @RainbowMango

**Comment by RainbowMango** [comment]
Here is the guideline for you:
https://github.com/karmada-io/karmada/blob/master/docs/contributors/guide/github-workflow.md

**Comment by RainbowMango** [comment]
After you push your code to `wongearl:feature/fix-karmada-descheduler-deploy`, we can re-open this PR.

**Comment by wongearl** [comment]
@RainbowMango thank you, I will follow "contributors/guide/github-workflow.md" when I find other bug.

**Pull request closed without merging**